src/styles/reset.css

```diff
@@ -0,0 +1,8 @@
+body,
+.card__name,
+.card__id,
+.price__number,
+.price__name,
+.rating__number {
+  margin: 0;
+}

```

Comment:
Not a big problem, but you could reset margins by simply using tag names here, not class names that refer to another file (according to their prefixes)